### DISCUSSION TITLE
Welcome to Enterprise Gateway Discussions!

### DISCUSSION THREAD

**Discussion by kevin-bates** [jupyter/enterprise_gateway#912] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kjdoyle** [comment]
Hi All. My name is Kevin Doyle and I work on a product called IBM Spectrum Conductor which utilizes Jupyter Enterprise Gateway. I have worked on contributions for the Conductor process proxy in JEG and am looking to do more for extending the usage of JEG in the product. Looking forward to working with you all more.

**Reply by kevin-bates** [reply]
Hi Kevin - Happy New Year! and thanks for the opening discussion entry.

I stumbled upon this feature and thought we (EG) should see how well this works since many of the new issues tend to be questions.

Let's see how this goes.
